R/deployApp.R

```diff
@@ -298,27 +303,17 @@ deployApp <- function(appDir = getwd(),
         bundle <- client$uploadApplication(application$id, bundlePath)
       }
     })
-  } else {
-    # redeploy current bundle
-    bundle <- application$deployment$bundle
-  }
 
-  # write a deployment record only if this is the account that owns the content
-  if (is.null(application$owner_username) ||
```

Comment:
When you're deploying to content that you don't own (but still have permission to update), you still want to update the `lastSyncTime`, bundle id` etc.